src/Network/Network/LoadBalancer/SetAzureLoadBalancerBackendPool.cs

```diff
@@ -0,0 +1,211 @@
+        [ValidateNotNullOrEmpty]
+        public string ResourceId { get; set; }
+
+        [Parameter(
+           Mandatory = false,
+           HelpMessage = "Do not ask for confirmation.")]
+        public SwitchParameter Force { get; set; }
+
+        [Parameter(Mandatory = false)]
+        public SwitchParameter PassThru { get; set; }
+
+        public override void Execute()
+        {
+            base.Execute();
+
+            BackendAddressPool backendAddressPool = null; 
+
+            if (this.IsParameterBound(c => c.LoadBalancer))
+            {
+                this.ResourceGroupName = this.LoadBalancer.ResourceGroupName;
+                this.LoadBalancerName = this.LoadBalancer.Name;
+            }
+            
+            if (this.IsParameterBound(c => c.ResourceId))
+            {
+                ResourceIdentifier resourceIdentifier = new ResourceIdentifier(this.ResourceId);
+                this.ResourceGroupName = resourceIdentifier.ResourceGroupName;
+                this.LoadBalancerName = resourceIdentifier.ParentResource.Split('/')[1];
+                this.Name = resourceIdentifier.ResourceName;
+            }
+
+            if (this.IsParameterBound(c => c.InputObject))
+            {
+                ResourceIdentifier resourceIdentifier = new ResourceIdentifier(this.InputObject.Id);
+                this.ResourceGroupName = resourceIdentifier.ResourceGroupName;
+                this.LoadBalancerName = resourceIdentifier.ParentResource.Split('/')[1];
+                this.Name = resourceIdentifier.ResourceName;
+                backendAddressPool = SetupBackendPoolWithInputObject();
+            }
+
+            BackendAddressPool existingloadBalancerBackendAddressPool = null;
+
+            // Confirm if resource already exists
+            try
+            {
+                existingloadBalancerBackendAddressPool = this.NetworkClient.NetworkManagementClient.LoadBalancerBackendAddressPools.Get(this.ResourceGroupName, this.LoadBalancerName, this.Name);
+            }
+            catch (Rest.Azure.CloudException exception)
+            {
+                if (exception.Response.StatusCode == System.Net.HttpStatusCode.NotFound)
+                {
+                    existingloadBalancerBackendAddressPool = null;
+                }
+                else
+                {
+                    throw;
+                }
+            }
+
+            // Return error message to user if loadbalancer pool doesnt exist
+            if (existingloadBalancerBackendAddressPool == null)
+            {
+                throw new Exception(string.Format("A BackendPool with name '{0}' in resource group '{1}' not found under LoadBalancer '{2}'. Please use New-AzLoadBalancerBackendAddressPool to create a BackendPool .",
+                    this.Name, this.ResourceGroupName, this.LoadBalancerName));
+            }
```

Comment:
we can move lines 128 to 132 to line 119

Reply:
Cleaned it up to accommodate this, thanks